### PR TITLE
 4_naidendev

### PR DESCRIPTION
**Pull request by naidendev**

### CODE REVIEW

**Comment by oddcommits** [comment]
resolve conflicts first, @naidendev

File: README.md [hunk 1]
```diff
@@ -89,3 +89,4 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 - [John Doe](./john-doe/README.md)
 - [Sebastian Harvey](./jsstar/README.md)
 - [White Collin](./whitecollin/)
+- [naidendev](./naidendev/README.md)
```
Comment (oddcommits) [inline README.md:92]: write your full name instead of nickname

File: naidendev/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+About Me
```
Comment (oddcommits) [inline naidendev/README.md:1]: `# About Me` will make more sense

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -89,6 +89,7 @@ This repository is designed to help newbies practice using Git and GitHub by cre
...
```
Comment (oddcommits) [inline README.md:92]: put your name at the end of contributors

File: naidendev/README.md (hunk 2 above)
Comment (oddcommits) [reply naidendev/README.md:1]: why don't fix this?

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (naidendev) [reply README.md]: fixed it

File: naidendev/README.md (hunk 2 above)
Comment (naidendev) [reply naidendev/README.md]: fixed it

File: README.md (hunk 3 above)
Comment (naidendev) [reply README.md]: fixed it

**Comment by oddcommits** [comment]
resolve conflicts, @naidendev

**Review by oddcommits — APPROVED** [review]
looks good

**Pull request merged**